docs/quickstarts/packet.md

```diff
@@ -5,262 +5,311 @@
-Flatcar Container Linux using Route53 as the DNS provider.
+This guide shows how to create a Lokomotive cluster on [Packet](https://www.packet.com/). By the
+end of this guide, you'll have a basic Lokomotive cluster running on Packet with a demo application
+deployed.
 
-By the end of this guide, you'll have a production-ready Kubernetes cluster running on Packet.
+The guide uses `t1.small.x86` as the Packet device type for all created nodes. This is also the
+default device type.
 
-## Requirements
+Lokomotive runs on top of [Flatcar Container Linux](https://www.flatcar-linux.org/). This guide
+uses the `stable` channel.
 
-* Basic understanding of Kubernetes concepts.
-* Packet account, Project ID and auth token (sometimes also referred to as [User Level API
-  key](https://www.packet.com/developers/docs/API/getting-started/)).
-* AWS account and IAM credentials (optional for Route53 DNS configuration).
-* AWS Route53 DNS Zone (registered Domain Name or delegated subdomain).
-* Terraform v0.12.x and [terraform-provider-ct](https://github.com/poseidon/terraform-provider-ct) v0.5.0
-  installed locally.
-* Local BGP enabled. More information on how to enable Local BGP for the Packet Project is found in
-  the [Packet support document](https://support.packet.com/kb/articles/bgp).
-* An SSH key pair for management access.
-* `kubectl` installed locally to access the Kubernetes cluster.
+The guide uses [Amazon Route 53](https://aws.amazon.com/route53/) as a DNS provider. For more
+information on how Lokomotive handles DNS, refer to [this](../concepts/dns.md) document.
 
-## Steps
+[Lokomotive components](../concepts/components.md) complement the "stock" Kubernetes functionality
+by adding features such as load balancing, persistent storage and monitoring to a cluster. To keep
+this guide short you will deploy a single component - `httpbin` - which serves as a demo
+application to verify the cluster behaves as expected.
 
-### Step 1: Install lokoctl
+## Requirements
 
-lokoctl is a command-line interface for Lokomotive.
+* A Packet account with a project created and
+  [local BGP](https://www.packet.com/developers/docs/network/advanced/local-and-global-bgp/)
+  enabled.
+* A Packet project ID.
+* A Packet
+  [user level API key](https://www.packet.com/developers/docs/API/getting-started/)
+  with access to the relevant project.
+* An AWS account.
+* An AWS
+  [access key ID and secret](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html)
+  of a user with
+  [permissions](https://github.com/kinvolk/lokomotive/blob/master/docs/concepts/dns.md#aws-route-53)
+  to edit Route 53 records.
+* An AWS Route 53 zone (can be a subdomain).
+* An SSH key pair for accessing the cluster nodes.
+* Terraform `v0.12.x`
+  [installed](https://learn.hashicorp.com/terraform/getting-started/install.html#install-terraform).
+* The `ct` Terraform provider `v0.5.0`
+  [installed](https://github.com/poseidon/terraform-provider-ct#install).
```

Comment:
Should we link to https://github.com/poseidon/terraform-provider-ct/blob/v0.5.0/README.md#install?

Reply:
Good idea. Thanks!